src/js/smooch.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import pick from 'lodash.pick';
+import { batchActions } from 'redux-batched-actions';
 
 import { store } from './stores/app-store';
```

Comment:
it could affect the time to first render in theory if we also do a smart job on lazy loading the ui